proxy/proxy.go

```diff
@@ -52,6 +53,16 @@ func (p *Proxy) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		h = newHTTPProxy(t.URL, p.tr)
 	}
 
+	// strip the prefix, if the target was tagged properly
```

Comment:
I was digging through this, and it seems like the `Target` object could have a `Modify` method which accepts an `http.Request` object and modifies the request to its liking. In the case that we had more `modifier` tags, this could be quite useful